hazelcast/src/main/java/com/hazelcast/instance/Node.java

```diff
@@ -799,4 +801,12 @@ public BuildInfo getBuildInfo() {
         return attributes;
     }
 
+    private void logGroupPasswordInfo() {
+        if (! StringUtil.isNullOrEmpty(config.getGroupConfig().getPassword())) {
```

Comment:
Please add no whitespace after the `!`. Also a static import for `isNullOrEmpty()` can be used.
```java
if (!isNullOrEmpty(config.getGroupConfig().getPassword())) {
```